_projects/commit_based_perftest.md

```diff
@@ -1,5 +1,6 @@
 ---
-desc: "A technique to view the performance of a piece of software and automatically identify and classify performance drops and gains."
+desc: "A technique to view the performance of a piece of software and automatically identify and classify performance
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.